src/Doctrine/Annotation/Token.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Doctrine\Annotation;
+
+/**
+ * A Doctrine annotation token.
+ */
+class Token
+{
+    const T_NONE = 1;
```

Comment:
`T_STRING` is for tokens that are actually parsed as strings by Doctrine Annotations. In this exemple, the words would be ignored and treated just like whitespaces, but they are not whitespaces, so I think `T_WHITESPACE` is not suitable either.

Maybe I should just add a comment next to the constant to make things clearer?